locallib.php

```diff
@@ -440,7 +459,7 @@ public function get_views($query = '') {
                                       array('users' => array( array($field => $username,
                                                                     'query' => $query))));
             $result = array_pop($result);
-            $result['views']['ids'] = array_map('intval', explode(',', $result['views']['ids']));
+            $result['views']['ids'] = array_map('intval', explode(', ', $result['views']['ids']));
```

Comment:
This change breaks the submission page under certain circumstances, is there a specific reason this should be here?
If not we should remove it. 
```suggestion
            $result['views']['ids'] = array_map('intval', explode(',', $result['views']['ids']));
```

Reply:
Long time since I looked at this, and you've fixed and merged anyway (Thanks!), but yes this is a bug :-(